templates/layout.hbs

```diff
@@ -53,5 +55,7 @@
         {{~> page}}
     </div>
     {{> footer}}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/prism.min.js"></script>
```

Comment:
One quick question here – can you check the size of these two loads vs. using a custom build of highlight.js with *only* Rust support baked in? (I have no idea what the difference is there; this may still come out ahead!) If a custom highlight.js build is indeed smaller, we should use that and vendor it in here. In that case, we may have to pick a different theme, or we may be able to find a variant of this one tailored for that purpose.

Reply:
The total size of the added JS is about 5.4kb gzipped. So it is a bit of an overhead just for a few markup. I think that pre-processing the highlight on the serverside would be a better approach. 

I can try to integrate Syntect and see how it feels.

Reply:
after briefly looking over `Syntect` I can say that the integration will take me a long time because of my lack of Rust skills. I will take a look over `highlight.js` and see how we can use that.